.github/workflows/ci-lite-test-full.yml

```diff
@@ -32,15 +31,17 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, windows-2022, macOS-11]
-        python-version: ["3.7", "3.10"]  # minimum, maximum
```

Comment:
May I suggest that we add extra parameterizations in a separate PR? We don't need to make this one unnecessarily long

Reply:
it was suggested by @justusschock but also fine to add it in extra PR